media/cdm/BUILD.gn

```diff
@@ -170,6 +170,21 @@ source_set("cdm") {
     public_deps += [ "//third_party/fuchsia-sdk/sdk/fidl/fuchsia.media.drm:fuchsia.media.drm_hlcpp" ]
     deps += [ "//media/fuchsia/common" ]
   }
+
+  if (is_cobalt) {
+    import("//starboard/build/buildflags.gni")
+    sources += [
```

Comment:
This should be gated by `use_starboard_media` for all starboard and media related files.

Reply:
done